bref

```diff
@@ -66,20 +68,28 @@ $app->command('init', function (SymfonyStyle $io) {
     ][$choice];
 
     $fs = new Filesystem;
-    $io->writeln('Creating template.yaml');
-    $fs->copy(__DIR__ . "/template/$templateDirectory/template.yaml", 'template.yaml');
-    if ($fs->exists(__DIR__ . "/template/$templateDirectory/index.php")) {
-        $io->writeln('Creating index.php');
-        $fs->copy(__DIR__ . "/template/$templateDirectory/index.php", 'index.php');
+    $finder = new Finder();
+    $finder->files()->in("$baseTemplateDir/$templateDirectory");
+
+    foreach ($finder as $file) {
+        $fileName = $file->getRelativePathname();
+        $srcFile = "$baseTemplateDir/$templateDirectory/$fileName";
+        if ($fs->exists($srcFile)) {
+            $io->writeln("Creating $fileName");
+            $fs->copy($srcFile, $fileName);
+        }
```

Comment:
Thanks that's a good solution I think!

I started looking into whether we could avoid using the Finder, since it's a new dependency in `composer.json` now.

Something like `$files = scandir("$baseTemplateDir/$templateDirectory");` should be a simple equivalent? Then you can use `$files` in the `foreach`.

Reply:
I switched to a RecursiveDirectoryIterator to avoid the Finder component as dependency